lib/screens/v2/widget_instance/departures.ex

```diff
@@ -487,12 +487,12 @@ defmodule Screens.V2.WidgetInstance.Departures do
   def serialize_inline_alerts([first_departure | _]) do
     first_departure
     |> Departure.alerts()
-    |> Enum.filter(&alert_is_inline?/1)
+    |> Enum.filter(&alert_inline?/1)
     |> Enum.map(&serialize_inline_alert/1)
   end
 
-  defp alert_is_inline?(%{effect: :delay}), do: false
-  defp alert_is_inline?(_), do: false
+  defp alert_inline?(%{effect: :delay}), do: false
+  defp alert_inline?(_), do: false
```

Comment:
i think this would be `inline_alert?`, wdyt?

Reply:
Yep, I like that better too.